wireless_modules_py/sensors/wind_sensor.py

```diff
@@ -0,0 +1,77 @@
+from sensor_base import Sensor
+import serial
+import time
+
+
+class WindSensor(Sensor):
+    """
+    A class for an anemometer connected via serial port.
+    """
+
+    def __init__(self, port):
+        """
+        Initialise the anemometer to read wind direction and wind speed data.
+        :param port: The serial port that the anemometer is connected to.
+        """
+        self.baudrate = 19200
+        self.sensor = serial.Serial(port, self.baudrate)
+
+        self.readings = []
+
+        # track the last time read() was called
```

Comment:
Don't need this comment.

Reply:
Will remove this line.
Other than this line, were there any other comments in particular that you found redundant and should be removed?